plugins/status/plugin.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/open-policy-agent/opa/plugins"
 	"github.com/open-policy-agent/opa/plugins/bundle"
 	"github.com/open-policy-agent/opa/util"
+	prom "github.com/prometheus/client_golang/prometheus"
```

Comment:
[nit] It would be cool if the imports could be lumped into blocks: stdlib, third party, our own packages. As such, this should go into the block above.